src/ui/views/index.tsx

```diff
@@ -45,12 +46,14 @@ const App = ({ wallet }: { wallet: any }) => {
   return (
     <ThemeProvider theme={theme}>
       <WalletProvider wallet={wallet}>
-        <GlobalStyles
-          styles={{
-            body: { backgroundColor: '#121212' },
-          }}
-        />
-        <Main />
+        <NewsProvider>
```

Comment:
Is there a way to avoid too much providers ? 
It's fine for now, but it build up might be too many providers in here.

Reply:
I think if we had better state management it could be avoided.